### ISSUE TITLE
fix for PWMERGE_SUR example

### ISSUE THREAD

**Issue by rdlxxvii** [ltsp/ltsp#118]
@ https://ltsp.org/man/ltsp.conf/ a example for PWMERGE_SUR is shown, that does not work: 

> PWMERGE_SUR=".*${HOSTNAME#pc}"

I fixed it with a small patch to 54-pam.sh: https://gk7.condor-edv.com/nc2/s/pzPENmJPZr6F7qE

Now the example needs to be changed to 

> PWMERGE_SUR=".*%{HOSTNAME#pc}"

**Comment by alkisg** [comment]
Thank you, that was indeed in my TODO list. :)
I'll copy the patch here to merge it later, as LTSP is pretty much frozen now due to the Ubuntu 20.04 release.

```shell
--- 54-pam.sh	2020-02-27 11:43:33.714811245 +0100
+++ /usr/share/ltsp/client/init/54-pam.sh	2020-02-26 22:30:36.283346779 +0100
@@ -6,13 +6,16 @@
     re "$_LTSP_DIR/client/login/pwmerge" \
-        ${PWMERGE_SUR:+"--sur=$PWMERGE_SUR"} \
-        ${PWMERGE_SGR:+"--sgr=$PWMERGE_SGR"} \
-        ${PWMERGE_DUR:+"--dur=$PWMERGE_DUR"} \
-        ${PWMERGE_DGR:+"--dgr=$PWMERGE_DGR"} \
+        ${PWMERGE_SUR:+"--sur=$(re eval_percent "$PWMERGE_SUR")"} \
+        ${PWMERGE_SGR:+"--sgr=$(re eval_percent "$PWMERGE_SGR")"} \
+        ${PWMERGE_DUR:+"--dur=$(re eval_percent "$PWMERGE_DUR")"} \
+        ${PWMERGE_DGR:+"--dgr=$(re eval_percent "$PWMERGE_DGR")"} \
         -lq /etc/ltsp /etc /etc
```

**Comment by rdlxxvii** [comment]
Great, Thnx.

**Comment by alkisg** [comment]
Fix committed in 219f59b271ae8e760b5899db16c17edd82098864.
